README.md

```diff
@@ -176,23 +176,29 @@ assert(#{ a: 1, b: 2 } === #{ b: 2, a: 1 });
 
 `Record` and `Tuple` types are completely and deeply constant, if they have the same values stored, they will be considered strictly equal.
 
-[Each of the four JS equality algorithms](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) gives the same result when comparing records or tuples:
+`===` follows `SameValue` equality when comparing values inside `Record` or `Tuple`.
 
 ```js
-// strict equality
-assert(#{ a:  1} === #{ a: 1 });
+assert(#{ a:  1 } === #{ a: 1 });
 assert(#[1] === #[1]);
 
+assert(#{ a: -0 } !== #{ a: +0 });
+assert(#[-0] !== #[+0]);
+assert(#{ a: NaN } === #{ a: NaN });
+assert(#[NaN] !== #[NaN]);
```

Comment:
Either this line or the previous line is wrong, surely?

Reply:
copy paste typo, fixed! my keyboard betrays me.